es/lib/isEmpty.js

```diff
@@ -1,10 +1,38 @@
+function _typeof(obj) { if (typeof Symbol === "function" && typeof Symbol.iterator === "symbol") { _typeof = function _typeof(obj) { return typeof obj; }; } else { _typeof = function _typeof(obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; }; } return _typeof(obj); }
+
 import assertString from './util/assertString';
 import merge from './util/merge';
 var default_is_empty_options = {
-  ignore_whitespace: false
+  ignore_whitespace: false,
+  to_string: false
 };
 export default function isEmpty(str, options) {
-  assertString(str);
```

Comment:
Why remove the assertion?

Reply:
Moved it to a condition. Please read my comments in the PR